.github/workflows/ci.yml

```diff
@@ -301,8 +301,8 @@ jobs:
     name: Push Self Hosted Image
     needs: [ build-self-hosted, test ]
     secrets: inherit
-    if: ${{ github.event_name == 'push' && github.event.ref == 'refs/heads/main' && github.repository_owner == 'codecov' }}
-    uses: codecov/gha-workflows/.github/workflows/self-hosted.yml@v1.2.12
+    #if: ${{ github.event_name == 'push' && github.event.ref == 'refs/heads/main' && github.repository_owner == 'codecov' }}
```

Comment:
Just double checking we want this removed? Does this mean that forks will be able to release our images?

Reply:
No thats a testing change. I was testing one last thing prior to being ready for release. I should have made this a draft pr. 

Also fork still couldn't release, it would just fail unless they also had dockerhub creds for our docker org.